pkg/controller/pagerdutyintegration/clusterdeployment_created.go

```diff
@@ -114,10 +115,10 @@ func (r *ReconcilePagerDutyIntegration) handleCreate(pdclient pd.Client, pdi *pa
 		r.reqLogger.Info("Creating PD service", "ClusterID", pdData.ClusterID, "BaseDomain", pdData.BaseDomain)
 		_, createErr = pdclient.CreateService(pdData)
 		if createErr != nil {
-			localmetrics.UpdateMetricPagerDutyCreateFailure(1, ClusterID, pdi.Name)
+			localmetrics.UpdateMetricPagerDutyCreateFailure(1, pdData.ClusterID, pdi.Name)
```

Comment:
clusterID value doesn't change here. Could you give a justification for getting the value out of pdData instead of just using the var here?

Reply:
basically consistency. I do not like that the functions on top is getting its value from "pdData.ClusterID" and then just right below another functions getting ClusterID. those 2 hold the same value.